src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs

```diff
@@ -256,10 +254,14 @@ private static void PopulateComponentToFileMapForFile(JObject jDocument, string
             }
         }
 
-        private static JObject GetJsonDocumentForComponentsFile(string componentsJSONPath)
+        private static JObject GetJsonDocumentForComponentsFile(
+            GitHttpClient gitClient,
+            RoslynInsertionToolOptions options,
+            string componentsJSONPath)
         {
             JObject jsonDocument = null;
 
+            // TODO: use gitClient
             if (File.Exists(componentsJSONPath))
```

Comment:
just curious - what is this stuff doing and why do we not care about it yet (I'm presuming the file will never exist now)

Reply:
I believe what it's actually doing is downloading VSSetup for the build we're inserting (about 200MB) and reading a few config files from it. see #572

Reply:
This whole method can be removed. we can just look up the version file from the build that we find

Reply:
Pushing this off to a future PR